src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -0,0 +1,77 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoGenerator {
+    private final int purchasePrice;
+    private final int purchaseAmount;
+
+    public LottoGenerator(int purchaseAmount) {
+        this.purchaseAmount = purchaseAmount;
+        this.purchasePrice = 1000;
```

Comment:
```suggestion
        this.purchasePrice = 1000;
```

객체마다 `1000` 고정값을 가질 필요는 없을 것 같아요.
상수로 치환시키는 것이 좋을 것 같습니다